scripts/build_example

```diff
@@ -55,7 +55,9 @@ bazel_build_flags+=(
 
 # Use a different output_base so that we don't lose incremental state.
 # See https://docs.bazel.build/versions/master/command-line-reference.html#flag--output_base.
-bazel --output_base="$CACHE_DIR/client" build "${bazel_build_flags[@]}" "//examples/$TARGET/client"
+bazel --output_base="$CACHE_DIR/client" build "${bazel_build_flags[@]}" \
+    "//examples/$TARGET/client" \
+    "//oak/common:app_config_serializer"
```

Comment:
Why is this added here?

Reply:
It will be used at some point to create configuration files for examples.'
Deleted.